packages/main/src/Bar.ts

```diff
@@ -65,6 +67,16 @@ class Bar extends UI5Element {
 	@property()
 	design: `${BarDesign}` = "Header";
 
+	/**
+	 * Used to define the role of the bar.
+	 * @private
+	 * @default "Toolbar"
+	 * @since 2.9.0
+	 *
+	 */
+	@property()
+	accessibleRole: `${BarAccessibleRole}` = "Toolbar";
```

Comment:
We'll need to provide recommendations to the users on how to use this new property.
In the openui5 project the logic is internal and no explanations were required.
Also in order to properly test this ourselves we could provide a dev sample that changes the number of interactive elements on runtime and therefore changes the role accordingly.

Reply:
I've added additional documentation and example. But i've added a static example with just two bars explaining the both cases, because we don't provide dynamic change of the role, so such example wont be needed.